cloudflare/resource_cloudflare_zone.go

```diff
@@ -47,6 +47,11 @@ func resourceCloudflareZone() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"plan": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"Free Website", "Business Website", "Pro Website", "Enterprise Website"}, false),
```

Comment:
Would be better to use `free`, `pro`, `business`, `enterprise` here and convert internally between them and API view. e.g.
`free` <=> `Free Website`

Reply:
Apologies @patryk but I think there is a little bit of confusion here as an earlier comment mentioned we don't want to apply any fuzzy matches to the values and instead you wanted the full values. I might have misunderstood your intentions so would you mind clarifying again to avoid extra back and forth? Perhaps a patch demonstrating what changes you intend here?

Reply:
@patryk please let us know... this is just a small change compared to my next PR 😄

Reply:
Let me summarize to see if I got this: you'd still be ok to use the ID as they are now, but instead of maping them with their descriptive names in the HCL and docs you'd prefer to show the legacy IDs to the user?
This means keeping a 1:1 match between the descriptive name of the plan and the legacy ID, like a `map[string]string` in a private const?

Reply:
Yes, this is what I meant. In the future we will clean this mess up, but IDs will stay the same. Thanks!